basics.rb

```diff
@@ -8,8 +8,9 @@
 post '/payload' do
   push = JSON.parse(request.body.read)
   puts "I got some JSON: #{push.inspect}"
-  jsontest = "I got some JSON: #{push.inspect}"
-  Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927
+
+  Collaborator.addByIssue repo_name: push["repository"]["full_name"].to_s, issue_num: push["issue"]["number"].to_i, user_login: push["issue"]["user"]["login"].to_s
+
   Collaborator.add repo_name: "githubschool/on-demand-github-pages", issue_num: 1
```

Comment:
@stoe I think we would need to keep `Collaborator.add repo_name: "githubschool/open-enrollment-classes-introduction-to-github", issue_num: 927` as well for the other repository.